test/e2e_node/memory_manager_test.go

```diff
@@ -324,7 +324,7 @@ var _ = SIGDescribe("Memory Manager [Disruptive] [Serial] [Feature:MemoryManager
 		if *is2MiHugepagesSupported {
 			ginkgo.By("Configuring hugepages")
 			gomega.Eventually(func() error {
-				return configureHugePages(hugepagesSize2M, hugepages2MiCount)
+				return configureHugePages(hugepagesSize2M, hugepages2MiCount, pointer.IntPtr(0))
```

Comment:
Instead of assuming Numa to be at Node 0 always; should we not check if 
A) Numa node is set, if it is set use `getAllNUMANodes()` to get the first index or any index.
B) if it is not set use, nil so that the huge-pages test can go as it is.

Reply:
It will require some change under tests(because we have some assumptions regarding the NUMA node), the only case when it will not work is when the node has multiple NUMA nodes and node0 will not have memory at all, but I think it is a very rare situation for CI environments.

Reply:
thanks! This is mostly for my understanding; what kind of assumptions do we have about the NUMA node.